docs/interactive/launchbuttons.ipynb

```diff
@@ -118,7 +118,7 @@
    "metadata": {},
    "source": [
     "(launch/thebelab)=\n",
-    "## Live interactive pages with Thebelab\n",
+    "## Live interactive pages with ThebeLab\n",
```

Comment:
note to self: we need to replace all use of the word `thebelab` with `thebe`

Reply:
I was going to do this, but then I realised I am not sure which replacements I can and cannot do (because I might be breaking things). Does it suffice to replace the occurrences in the prose?

Reply:
yep that should be fine - we can take a pass in the codebase if need be later on, but most of that heavy lifting is done by `sphinx-thebe` anyway. Thanks for offering to do this :-)